apps/openmw/mwrender/sky.cpp

```diff
@@ -1576,30 +1570,21 @@ bool SkyManager::isEnabled()
     return mEnabled;
 }
 
-bool SkyManager::hasRain()
+bool SkyManager::hasRain() const
 {
     return mRainNode != nullptr;
 }
 
-void SkyManager::update(float duration)
+float SkyManager::getEffectFade() const
```

Comment:
Now `mEffectFade` has a getter and is part of the public API for this class, it makes sense to give it a more illustrative name, or at least give the getter a name that makes it clear it's the rain intensity value that it represents. Other than that, LGTM.